ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -80,46 +79,34 @@ public void shouldUseAndCloseTestExecutor() throws Exception {
   }
 
   @Test
-  public void shouldFailWithIncorrectArgs() throws UnsupportedEncodingException {
-
-    // Given:
-    System.setSecurityManager(new TestSecurityManager());
-
+  public void shouldFailWithIncorrectTest() throws IOException {
     // When:
-    try {
-      KsqlTestingTool.loadAndRunTests(new String[]{"foo"});
-    } catch (final Exception e) {
-      assertThat(e, instanceOf(TestSecurityManager.ExitSecurityException.class));
-      final TestSecurityManager.ExitSecurityException exitSecurityException = (TestSecurityManager.ExitSecurityException) e;
-      assertThat(exitSecurityException.getStatus(), equalTo(-1));
-      assertThat(errContent.toString("UTF-8"), equalTo("Failed to start KSQL testing tool: foo (No such file or directory)\n"));
-    }
-  }
+    KsqlTestingTool.runWithThripleFiles(
+        "src/test/resources/test-runner/incorrect-test/statements.sql",
+        "src/test/resources/test-runner/incorrect-test/input.json",
+        "src/test/resources/test-runner/incorrect-test/output.json");
 
+    // Then:
+    assertThat(errContent.toString("UTF-8"),
+        containsString("\t>>>>> Test failed: Expected <1001, 101> with timestamp=0 but was <101, 101> with timestamp=0\n"));
+  }
 
-  static class TestSecurityManager extends SecurityManager {
-
-    final class ExitSecurityException extends SecurityException {
-      private final int status;
+  @Test
+  public void shouldFailWithIncorrectArgs() throws UnsupportedEncodingException {
 
-      ExitSecurityException(final int status) {
-        this.status = status;
-      }
 
-      int getStatus() {
-        return this.status;
-      }
-    }
+  }
 
-    @Override
-    public void checkExit(final int status) {
-      if (status != 0) {
-        throw new TestSecurityManager.ExitSecurityException(status);
-      }
-    }
 
-    @Override
-    public void checkPermission(final Permission perm) {}
+  private void runTestCase(
+      final String statementsFilePath,
+      final String inputFilePath,
+      final String outputFilePath
+      ) throws IOException {
+    // When:
+    KsqlTestingTool.runWithThripleFiles(statementsFilePath, inputFilePath, outputFilePath);
 
+    // Then:
+    assertThat(outContent.toString("UTF-8"), containsString(">>> Test passed!"));
```

Comment:
```suggestion
    assertThat(outContent.toString("UTF-8"), containsString("Test passed"));
```
makes it more robust to small changes (same comment applies to the other statements that verify that strings)